### ISSUE TITLE
Cant create refs/heads/feature/branch

### ISSUE THREAD

**Issue by samdnz** [octokit/octokit.js#2289]
### Please avoid duplicates

- [X] I checked [all open bugs](https://github.com/octokit/octokit.js/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and none of them matched my problem.

### Reproducible test case

https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgcwKYwPIGMYQNbDwC+cAZlBCHAEQACAhjsBAHYDOA9MoQBYCuAI2oAoUnxZNWcYGwDKMKMBbIAFAHce9GAC44bBUuQBKREWHnhWVvsRwAJlvpwSAXjj019QnAg58hADooVABHPlR9FWoAcQBRABU4DmojCyt2eAcYJzcPL3g0TD8CGCiUoIiYAO4qrGCtVAAlVFIVBGE4Tq7O4NJdal7OHlR6O05SEZg+YI4QAE8AWgmtadRqABoO7q62TX6AN3oAG2A7PU0Rba7gyH76FjmNrauINRZUKH6cAA8KyADXu8oJdtkQjABucwcDhwVDfMCoHDuOC9OC4OACVBwOqTVB2CzQ9w4PjHOCxKAUT5wZoTYISLF8MBZLGkLxHPFwI4QZBoM5qXhwFgQOAgaBYpSkCDCIA

### Please select the environment(s) that are relevant to your bug report

- [X] TypeScript
- [ ] Enterprise
- [ ] Browsers
- [X] Node
- [ ] Deno

### Version

"@actions/github": "^5.0.3"

### What happened?

When using the octokit via the actions package to create branches only names with 2 slashes are accepted. 

refs/heads/feature will work

but refs/heads/feature/my-feature will not 

### Would you be interested in contributing a fix?

- [ ] yes

**Comment by oscard0m** [comment]
Hi @samdnz,

Thanks for taking the time and effort on reporting the issue. Is very valuable that you share a TypeScript Playground.

I have some questions for you to try to get closer to the root cause of the issue:
- Would it be possible to get the output error you are getting?

I'm guessing you are getting some kind of error when trying to run your command.

Thanks

**Comment by samdnz** [comment]
```
creating refs/heads/acceptance/deploy-action-yaml from e0ff8a72cb53368408223a0426d6356aa327cc4c, pipeline-actions samdnz
Error: Reference update failed
```

with ref "refs/heads/acceptance-deploy-action-yaml" this would work

**Comment by timrogers** [comment]
@samdnz I've just tried this locally using the same code as you and I am able to create a branch with a name with a slash in it (e.g. `timrogers/foo`).

Could you check your `package-lock.json` and share the versions of all of the Octokit packages you're using?

Could you also share a working code sample for a branch you are able to create?

**Comment by samdnz** [comment]
https://github.com/samdnz/test-branch-create/runs/8157402605?check_suite_focus=true

everything is in this repo. the octokit dependency is transitive from the actions package

**Comment by timrogers** [comment]
This is rather strange! Would you be able to try using Octokit directly instead? I'm wondering if there is a fix in more recent Octokit versions.

**Comment by samdnz** [comment]
Yes this is rather strange. I have spent the better part of the week convincing myself that this was actually the cause of my issue.
